### PR TITLE
Parallel calls to git when getting the opam state

### PR DESCRIPTION
**Pull request by thomasblanc**
This saves about 2 minutes on a 15 minutes check.

I may have reindented things, sorry if this is an inconvenience. (Though I'm planning on rearranging the layout soon anyway).

The first commit adds the right dependencies in the opam description.

### CODE REVIEW

File: camelus_lib.ml [hunk 1]
```diff
@@ -524,52 +567,52 @@ module FormatUpgrade = struct
...
-    in
-    RepoGit.git gitstore ~env
-      ("commit-tree" ::
-       "-m" :: message ::
-       (if merge then ["-p"; onto] else []) @
-       [ "-p"; head;
-         tree ])
-    >|= String.trim
-    >|= fun hash -> Some (hash, message)
+      let%lwt _ =
```
Comment (AltGr) [inline camelus_lib.ml:570]: Please upgrade your `ocp-indent` :)

Comment (thomasblanc) [reply camelus_lib.ml:570]: :-)

Comment (AltGr) [reply camelus_lib.ml:570]: (or check that you have `strict_else=auto` configured)

Comment (AltGr) [reply camelus_lib.ml:570]: My bad, I wasn't using default ocp-indent config and overlooked adding the proper `.ocp-indent` file. Fixing my mess and pushing to the 2.0 branch :)

**Comment by AltGr** [comment]
Pushed to master by hand. Thanks!

**Pull request closed without merging**